www/js/settings.json

```diff
@@ -1,36 +1,27 @@
 {
-  "app": {
-    "title": "App Settings",
+  "slide-layout": {
+    "title": "Slide Layout",
     "settings": {
-      "theme": {
-        "title": "",
-        "type": "radio",
-        "options": {}
-      },
-      "themebg": {
-        "title": "",
-        "type": "dynamic",
-        "options": {}
-      },
-      "layout": {
-        "title": "App Layout",
+      "display-options": {
+        "title": "Display Options",
         "type": "switch",
         "options": {
-          "presenter-view": "Presenter View"
+          "akhandpaatt": {
+            "label": "Akhand Paatth View (Beta)",
+            "subLabel": "(Google ChromeCast Unavailable)"
+          },
+          "larivaar": "Larivaar",
+          "left-align": "Left-Align",
+          "autoplay": "Auto Play",
```

Comment:
this is a nit but can this option be closer to the autoplay delay slider?

Reply:
Okay